frontend/packages/dev-console/src/components/projects/CreateProjectListPage.tsx

```diff
@@ -11,6 +13,18 @@ export interface CreateProjectListPageProps extends ProjectListPageProps {
   onCreate?: (project: K8sResourceKind) => void;
 }
 
+export const CreateAProjectButton = (openProjectModal) => {
+  const { t } = useTranslation();
+  return (
+    <Trans t={t} ns="devconsole">
+      or{' '}
+      <Button isInline variant="link" onClick={openProjectModal}>
+        create a Project
+      </Button>
+    </Trans>
+  );
+};
+
```

Comment:
Below in `CreateProjectListPage` the `openProjectModal` callback is created and passed to the `CreateProjectListPage` `children`...

Just an idea, don't know if this works fine. You might be also can check the flag here and pass the button as optional the child? Wdyt?

```tsx
const canCreateProject = useFlag(FLAGS.CAN_CREATE_PROJECT);
const orProjectButton = canCreateProject && (
    <Trans t={t} ns="devconsole">
      {' or '}
      <Button isInline variant="link" onClick={openProjectModal}>
        create a Project
      </Button>
    </Trans>
)
```

and then:

```tsx
  return (
    <ProjectListPage {...props} title={title}>
      {children({ orProjectButton })}
    </ProjectListPage>
  );
```

Just an idea. If you prefer the current impl. that's fine for me. But we need to address the translation issues. (Unnecessary space and the full stop should be part of the translations.)